Update team tests

nixocio opened the pull request:
Update team tests

acruzgon commented:
hi @nixocio i saw this error and fixed it in here https://github.com/ansible/ansible-ui/pull/2553
i replaced `containsBy` with `getByDataCy` because it's more reliable, let me know if that helped

nixocio commented:
> hi @nixocio i saw this error and fixed it in here #2553 i replaced `containsBy` with `getByDataCy` because it's more reliable, let me know if that helped

Thanks @acruzgon that helps.

nixocio commented:
I am closing this one in favor of the other mentioned fix.
